src/all.js

```diff
@@ -1,10 +1,12 @@
 import uniswap_v2 from './exchanges/uniswap_v2'
 import pancakeswap from './exchanges/pancakeswap'
+import cryptoswap from './exchanges/cryptoswap'
```

Comment:
Please remove this for now. I will need to add a strategy first how to decide which route is more performant if the DEX is on the same blockchain (which is currently missing).

So by removing this, we could merge this PR without waiting for the strategy to be done. 

Once the strategy is ready, I can just activate cryptoswap by adding it here again.